backend/dl/detection.py

```diff
@@ -66,4 +67,17 @@ def detection_img_drive(IMAGE_UPLOAD_FOLDER, detection_type, problem_type, trans
         im_b64, label_set = rekognition_detection(image, problem_type)
     elif (detection_type == "yolo"):
         im_b64, label_set = yolo_detection(image)
+    if label_set:
+        backend_dir = (
+            ""
+            if (os.getcwd()).split("\\")[-1].split("/")[-1] == "backend"
+            else "backend"
+            if ("backend" in os.listdir(os.getcwd()))
+            else "../backend"
+        )
+        keys = label_set[0].keys()
+        with open(os.path.join(backend_dir, "detection_results.csv"), 'w', newline = '') as f:
```

Comment:
any particular reason this logic lives in `transform_image()` function. Might want to have a separate helper function that writes the object detection results to the appropriate backend path @Dashrekker

Reply:
just added the separate helper function as write_to_csv